build/ci/evergreen.yml

```diff
@@ -0,0 +1,338 @@
+          ATTRS: ${attr}
+        binary: make
+        args:
+          - otel
+tasks:
+  - name: compile
+    tags: ["code_health"]
+    commands:
+      - func: "build"
+  - name: binary_e2e
+    tags: ["e2e","required"]
+    must_have_test_results: true
+    commands:
+      - func: "install gotestsum"
+      - command: shell.exec
+        type: test
+        params:
+          <<: *go_options
+          shell: bash
+          script: |
+            set -Eeou pipefail
+            mkdir "$XDG_CONFIG_HOME/atlascli"
+            cat <<EOF > "$XDG_CONFIG_HOME/atlascli/config.toml"
+            [e2e]
+              org_id = "5e429e7706822c6eac4d5971"
+              public_api_key = "AAUMGJXA"
+              service = "cloud"
+            EOF
+      - func: "e2e test"
+        vars:
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_SERVICE: cloud
+          E2E_TAGS: cli
+  - name: atlas_kubernetes_generate_e2e
+    tags: ["e2e","atlas","kubernetes", "assigned_to_jira_team_cloudp_kubernetes_atlas"]
+    must_have_test_results: true
+    commands:
+      - func: "install gotestsum"
+      # - func: "e2e test"
+      #   vars:
+      #     MCLI_ORG_ID: ${atlas_org_id}
+      #     MCLI_PROJECT_ID: ${atlas_project_id}
+      #     MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+      #     MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+      #     MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+      #     MCLI_SERVICE: cloud
+      #     E2E_TAGS: atlas,cluster,kubernetes,generate
+  - name: atlas_kubernetes_apply_e2e
+    tags: [ "e2e","atlas","kubernetes", "assigned_to_jira_team_cloudp_kubernetes_atlas" ]
+    must_have_test_results: true
+    commands:
+      - func: "install gotestsum"
+      # - func: "e2e test"
+      #   vars:
+      #     MCLI_ORG_ID: ${atlas_org_id}
+      #     MCLI_PROJECT_ID: ${atlas_project_id}
+      #     MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+      #     MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+      #     MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+      #     MCLI_SERVICE: cloud
+      #     E2E_TAGS: atlas,cluster,kubernetes,apply
+  - name: atlas_kubernetes_install_e2e
+    tags: [ "e2e","atlas","kubernetes", "assigned_to_jira_team_cloudp_kubernetes_atlas" ]
+    must_have_test_results: true
+    commands:
+      - func: "install gotestsum"
+      # - func: "e2e test"
+      #   vars:
+      #     MCLI_ORG_ID: ${atlas_org_id}
+      #     MCLI_PROJECT_ID: ${atlas_project_id}
+      #     MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+      #     MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+      #     MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+      #     MCLI_SERVICE: cloud
+      #     E2E_TAGS: atlas,cluster,kubernetes,local
+  - name: snyk_monitor
```

Comment:
Can we either remove this or the github action that does the same thing?

Reply:
thanks! removed